modules/ossm-excluding-namespaces-from-cluster-wide-mesh-cli.adoc

```diff
@@ -5,8 +5,13 @@
 [id="ossm-excluding-namespaces-from-cluster-wide-mesh-cli_{context}"]
 = Including and excluding namespaces from a cluster-wide mesh by using the CLI
 
-By default, the {SMProductName} Operator uses discovery selectors to identify the namespaces that make up the mesh. Namespaces that do not contain the label defined in the `ServiceMeshMemberRoll` resource are not matched by the discovery selector and are excluded from the mesh.  
- 
+In a cluster-wide mesh, you can add discovery selectors to the `ServiceMeshControlPlane` resource. Discovery selectors define which namespaces the control plane can discover. The control plane ignores any namespace that does not match one of the discovery selectors, which excludes the namespace from the mesh.
+
+[NOTE]
+====
+If you install ingress or egress gateways in the control plane namespace, then you must include the control plane namespace in the discovery selectors.
```

Comment:
ISG prefers that you avoid "If... then" constructions unless you really need them to show a conditional relationship between two long clauses. 

```suggestion
If you install ingress or egress gateways in the control plane namespace, you must include the control plane namespace in the discovery selectors.
```

Reply:
done. also made change on console task.